### ISSUE TITLE
Unable to run `make bootstrap`

### ISSUE THREAD

**Issue by venkatramachandran** [kickstarter/android-oss#6]
Environment: Ubuntu 14.04.1
While running `make bootstrap`, I get the following error:

```
vagrant@vagrant:~/android-oss$ make bootstrap
git submodule sync --recursive
Synchronizing submodule url for 'vendor/material-design-icons'
Synchronizing submodule url for 'vendor/native-secrets'
git submodule update --init --recursive || true
Cloning into 'vendor/native-secrets'...
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Clone of 'git@github.com:kickstarter/native-secrets.git' into submodule path 'vendor/native-secrets' failed
git submodule foreach git checkout 
Entering 'vendor/material-design-icons'
# Copy java secrets over, and fallback to the example secrets if they don't exist.
cp vendor/native-secrets/android/Secrets.java app/src/main/java/com/kickstarter/libs/utils/Secrets.java \
		|| cp app/src/main/java/com/kickstarter/libs/utils/Secrets.java.example app/src/main/java/com/kickstarter/libs/utils/Secrets.java
cp: cannot stat ‘vendor/native-secrets/android/Secrets.java’: No such file or directory
cp vendor/native-secrets/ruby/secrets.rb lib/milkrun/secrets.rb || true
cp: cannot stat ‘vendor/native-secrets/ruby/secrets.rb’: No such file or directory
cp vendor/native-secrets/fonts/ss-kickstarter.otf app/src/main/assets/fonts/ss-kickstarter.otf || true
cp: cannot stat ‘vendor/native-secrets/fonts/ss-kickstarter.otf’: No such file or directory
# Copy koala endpoint configs over, and fallback to examples if they don't exist.
mkdir -p app/src/debug/res/values/
mkdir -p app/src/main/res/values/
cp vendor/native-secrets/android/koala_endpoint_debug.xml app/src/debug/res/values/koala_endpoint.xml \
		|| cp config/koala_endpoint.xml.example app/src/debug/res/values/koala_endpoint.xml
cp: cannot stat ‘vendor/native-secrets/android/koala_endpoint_debug.xml’: No such file or directory
cp vendor/native-secrets/android/koala_endpoint.xml app/src/main/res/values/koala_endpoint.xml \
		|| cp config/koala_endpoint.xml.example app/src/main/res/values/koala_endpoint.xml
cp: cannot stat ‘vendor/native-secrets/android/koala_endpoint.xml’: No such file or directory
# Copy over web client
cp -rf vendor/native-secrets/android/WebViewJavascriptInterface.java app/src/main/java/com/kickstarter/libs/WebViewJavascriptInterface.java \
		|| cp -rf config/WebViewJavascriptInterface.java app/src/main/java/com/kickstarter/libs/WebViewJavascriptInterface.java
cp: cannot stat ‘vendor/native-secrets/android/WebViewJavascriptInterface.java’: No such file or directory
cp -rf vendor/native-secrets/android/KSWebViewClient.java app/src/main/java/com/kickstarter/services/KSWebViewClient.java \
		|| cp -rf config/KSWebViewClient.java app/src/main/java/com/kickstarter/services/KSWebViewClient.java
cp: cannot stat ‘vendor/native-secrets/android/KSWebViewClient.java’: No such file or directory
./script/bootstrap
./script/bootstrap: 3: ./script/bootstrap: function: not found

./script/bootstrap: 6: ./script/bootstrap: Syntax error: "}" unexpected
make: *** [bootstrap] Error 2
```

What am I missing here?

**Comment by rahultailor** [comment]
I am also having same issue.

**Comment by wuzhenda** [comment]
It seems kickstarter has replace some trully used java file with false java file. but the Makefile was not update .

**Comment by luoser** [comment]
will be fixed once #11 merges!

**Comment by rome753** [comment]
on my Ubuntu 16, edit the ./script/bootstrap file with a text editor,
change "function say {..." to "say() {...",
and also "function loudspeaker {..." to "loudspeaker() {...",
then apt install ruby and curl,
then ok.
